FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,47 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+
+			switch result {
+			case .failure:
+				completion(.failure(Error.connectivity))
+
+			case let .success((data, response)):
+				guard let images = FeedImagesMapper.map(data: data, response: response) else {
+					return completion(.failure(Error.invalidData))
+				}
+				completion(.success(images))
+			}
+		}
+	}
+}
+
+final class FeedImagesMapper {
+	private init() {}
+
+	struct Root: Decodable {
```

Comment:
`Root` is not used outside of `FeedImagesMapper`, so it should be `private`.
```suggestion
	private struct Root: Decodable {
```

Reply:
👍